src/npc.h

```diff
@@ -552,6 +552,10 @@ class npc : public player
  npc &operator=(const npc &) = default;
  npc &operator=(npc &&) = default;
  ~npc() override;
+
+    /** Construct an NPC with standard stats */
+    static npc standard();
```

Comment:
Possibly `make_standard`? I want to start using it more where we have fake NPC's as opposed to manually constructing them.